packages/database/src/migrations/20210728020128-AddFQSSurveySyncGroups-modifies-data.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+import { generateId, insertObject, deleteObject } from '../utilities';
+import {
+  FQS1_CODE_MAPPING,
+  FQS2_CODE_MAPPING,
+} from './migrationData/20210728020128-AddFQSSurveySyncGroups;
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const KOBO_SURVEYS = [
+  {
+    code: 'FQS1',
+    entityQuestionCode: 'FQS1Primary_location/FQS1PrimarySchool',
+    questionCodeMapping: FQS1_CODE_MAPPING,
+    internalSurveyCode: 'LESMIS_FQS1',
```

Comment:
I'd like to keep some conceptual consistency with the other data source records, where the base `code` is internal. Is your worry here that we might sync multiple external surveys into a single internal survey?

```suggestion
    code: 'LESMIS_FQS1',
    entityQuestionCode: 'FQS1Primary_location/FQS1PrimarySchool',
    questionCodeMapping: FQS1_CODE_MAPPING,
    externalSurveyCode: 'FQS1',
```

Reply:
Oh ok, I see the comment further down now - looks like the base `code` is just for the specific record, and both the internal and external codes are inside the config?